sync15-adapter/src/util.rs

```diff
@@ -0,0 +1,135 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use std::convert::From;
+use std::time::Duration;
+use std::{fmt, num};
+use std::str::FromStr;
+use openssl;
+use base64;
+
+pub fn base16_encode(bytes: &[u8]) -> String {
```

Comment:
I'm surprised you had to hand-roll this, but :+1:

Reply:
I wrote most of a crate for this over the weekend, mainly since [rust-hex](https://github.com/KokaKiwi/rust-hex) (which I found out about after writing this) is much much slower (and it's hard to optimize given that it's API is based on `std::io::{Read,Write}`). But yeah I had to ask in the rust IRC if there was a builtin way and apparently there is not :(